apps/namada-interface/src/App/Staking/NewBondingPosition/NewBondingPosition.tsx

```diff
@@ -89,21 +91,21 @@ export const NewBondingPosition = (props: Props): JSX.Element => {
   // unbonding amount and displayed value with a very naive validation
   // TODO (https://github.com/anoma/namada-interface/issues/4#issuecomment-1260564499)
   // do proper validation as part of input
-  const amountToBondNumber = Number(amountToBond);
+  const amountToBondNumber: BigNumber = new BigNumber(amountToBond);
```

Comment:
I think it's just my fear of type inference :laughing: At least coming from Haskell where type inference is also supported, I prefer to add type annotations because I find the code easier to read and can also check the type against what the compiler thinks it should be. But I agree it's not necessary here so I've removed it.